### PR TITLE
feature(EJ2-64707): Updated getting started section with reviewed content.

### PR DESCRIPTION
**Pull request by Deepika5698**
### Feature description

Need to include the reviewed content.

### Analysis and design

Need to include the reviewed content for getting started section in ASP.NET Core and MVC Message component.
 
### Solution description
 
Added the reviewed content.
 
### Output screenshots
NA
 
### Areas affected and ensured
No area is affected due to this changes.
 
### Test cases
NA
 
### Test bed sample location
NA
 
### Additional checklist
 
* Did you run the automation against your implementation? NA
* Is there any API name change? No
* Is there any existing behavior change of other features due to this code change? No
* Does your new code introduce new warnings or binding errors? No
* Does your code pass all FxCop and StyleCop rules? Yes
* Did you record this case in the unit test or UI test? NA

### CODE REVIEW

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Add **Syncfusion.EJ2** namespace reference in `Web.config` under `Views` folder.
 
 ## Add style sheet
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:43]: Use below heading

Add a style sheet

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md [hunk 2]
```diff
@@ -24,15 +24,15 @@ This section briefly explains about how to include ASP.NET MVC Message control i
 
 ## Install ASP.NET MVC package in the application
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:25]: Use the below content
Install the ASP.NET MVC package in the application

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md [hunk 3]
```diff
@@ -24,15 +24,15 @@ This section briefly explains about how to include ASP.NET MVC Message control i
...
+Syncfusion ASP.NET MVC controls are available in [nuget.org.](https://www.nuget.org/packages?q=syncfusion.EJ2). Refer to the [NuGet packages topic](https://ej2.syncfusion.com/aspnetmvc/documentation/nuget-packages) to learn more about installing NuGet packages in various OS environments. To add ASP.NET MVC controls to the application, open the NuGet package manager in Visual Studio (Tools → NuGet Package Manager → Manage NuGet Packages for Solution) and search for [Syncfusion.EJ2.MVC5](https://www.nuget.org/packages/Syncfusion.EJ2.MVC5), then install it.
 
 > The Syncfusion.EJ2.MVC5 NuGet package has dependencies, [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) for JSON serialization and [Syncfusion.Licensing](https://www.nuget.org/packages/Syncfusion.Licensing/) for validating Syncfusion license key.
 
-> If you create ASP.NET MVC application with MVC4 package, search for [Syncfusion.EJ2.MVC4](https://www.nuget.org/packages/Syncfusion.EJ2.MVC4) and then install it. 
+> If you create ASP.NET MVC application with MVC4 package, search for [Syncfusion.EJ2.MVC4](https://www.nuget.org/packages/Syncfusion.EJ2.MVC4), then install it.
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:31]: remove end line empty space

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md [hunk 4]
```diff
@@ -74,7 +74,7 @@ In this getting started walk-through, the required scripts are referred using CD
 
 ## Register Syncfusion Script Manager
 
-Open `~/Views/Shared/_Layout.cshtml` page and register the script manager `EJS().ScriptManager()` at the end of `<body>` in the ASP.NET MVC application as follows. 
+Open the `~/Views/Shared/_Layout.cshtml` page and register the script manager `EJS().ScriptManager()` at the end of the `<body>` in the ASP.NET MVC application as follows.
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:77]: remove end line empty space

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/getting-started.md [hunk 5]
```diff
@@ -75,7 +75,7 @@ In this getting started walk-through, the required scripts are referred using CD
 
 ## Register Syncfusion Script Manager
 
-Open `~/Pages/Shared/_Layout.cshtml` page and register the script manager <ejs-script> at the end of `<body>` in the ASP.NET Core application as follows. 
+Open the `~/Pages/Shared/_Layout.cshtml` page and register the script manager <ejs-script> at the end of the `<body>` in the ASP.NET Core application as follows.
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/getting-started.md:78]: remove unwanted empty space

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md (hunk 1 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:43]: Replaced the heading.

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md (hunk 2 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:25]: Replaced the heading.

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md (hunk 3 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:31]: Removed the empty space.

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md (hunk 4 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:77]: Removed the empty space.

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/getting-started.md [hunk 6]
```diff
@@ -25,7 +25,7 @@ This section briefly explains about how to include ASP.NET Core Message control
 
 ## Install ASP.NET Core package in the application
```
[the remark this answers is not in the document]
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/getting-started.md:26]: Replaced with above heading.

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/getting-started.md (hunk 5 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/getting-started.md:78]: Removed the unwanted space.

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/getting-started.md [hunk 7]
```diff
@@ -25,7 +25,7 @@ This section briefly explains about how to include ASP.NET Core Message control
...
+Syncfusion ASP.NET Core controls are in [nuget.org.](https://www.nuget.org/packages?q=syncfusion.EJ2). Refer to the [NuGet packages topic](https://ej2.syncfusion.com/aspnetcore/documentation/nuget-packages/) to learn more about installing NuGet packages in various OS environments. To add ASP.NET Core controls to the application, open the NuGet package manager in Visual Studio (Tools → NuGet Package Manager → Manage NuGet Packages for Solution) and search for [Syncfusion.EJ2.AspNet.Core](https://www.nuget.org/packages/Syncfusion.EJ2.AspNet.Core/), then install it.
 
 > The Syncfusion.EJ2.AspNet.Core NuGet package has dependencies, [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) for JSON serialization and [Syncfusion.Licensing](https://www.nuget.org/packages/Syncfusion.Licensing/) for validating Syncfusion license key.
```
[the remark this answers is not in the document]
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/getting-started.md:31]: Replaced with above content.

**Review by PiramanayagamSyncfusion — APPROVED** [review]
Changes fine

**Pull request merged**